<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Activity Monitor</string>
  </property>
  <property name="stylesheet" stdset="0">
   <string notr="true">dark_style</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>289</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #5cb85c; color: white;</string>
    </property>
    <property name="text">
     <string>Start Monitoring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>289</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d9534f; color: white;</string>
    </property>
    <property name="text">
     <string>Stop Monitoring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #d9534f; color: white;</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>256</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
